TW 414 Refresh PagerDuty integration page (#4315)
postmanlabs/postman-docs: src/pages/docs/integrations/available-integrations/pagerduty.md

---

> PagerDuty is an incident management solution that integrates with monitoring stacks for alerting, on-call scheduling, and automatic escalation of critical incidents.

Reviewer: In the front matter, chop out page_id, warning, and order. Make sure to bump up the date.

This integration triggers incidents in PagerDuty based on your [Postman monitor results](/docs/monitoring-your-api/viewing-monitor-results/) so that your team can investigate and resolve collection run failures.

## Contents

* [Retrieving your PagerDuty Integration Key](#retrieving-your-pagerduty-integration-key)
* [Configuring Postman with PagerDuty](#configuring-postman-with-pagerduty)
* [Viewing PagerDuty results](#viewing-pagerduty-results)

> ## Retrieving your PagerDuty Integration Key

Reviewer: I don't think "Integration Key" needs to be capitalized. (Change globally)

> 1. Create an account or use a preexisting one to sign in to [PagerDuty](https://app.pagerduty.com/).

Reviewer 2: ```suggestion
1. Create an account or use an existing account to sign in to [PagerDuty](https://app.pagerduty.com/).
```

Reviewer: I'd almost say just cut this to "sign in to..." and if they don't have an account, that's kind of on them.

> 1. In PagerDuty, select **Services**. If you are creating a new service for this integration, select **Services** and then **+ New Service**.

Reviewer 2: ```suggestion
1. In PagerDuty, select **Services**. If you're creating a new service for this integration, select **Services** and then **+ New Service**.
```

Author: Thanks for catching this! Interesting that Vale didn't 👀

1. Enter a **Service Name**.

> 1. Using the **Integrations** dropdown, search and select **Postman**.

Reviewer 2: ```suggestion
1. Using the **Integrations** dropdown menu, search and select **Postman**.
```

Reviewer 2: Microsoft says to not use "dropdown" as a noun, but instead say "dropdown list" or "dropdown menu"

1. Select the **Add Service** link at the bottom of the page to create a new service.
1. Save the PagerDuty **Integration Key** to use later.

## Configuring Postman with PagerDuty

1. Select **[Integrations](https://go.postman.co/integrations)** from Postman's **[Home](https://go.postman.co/home)** page.

    <img alt="Navigate to Integrations" src="https://assets.postman.com/postman-docs/v10/select-integrations-from-home-v10.jpg" width="250px"/>

1. Search and select **PagerDuty**.
1. Select **Add Integration**.

    <img alt="Add an integration" src="https://assets.postman.com/postman-docs/v10/integrations-add-new-integration-v10.jpg" width="900px"/>

> 1. Enter the name of the integration, workspace, monitor, and your PagerDuty Integration Key, then select **Add Integration**.

Reviewer 2: ```suggestion
1. Enter the name of the integration, workspace, monitor, and your PagerDuty integration key, then select **Add Integration**.
```

Reviewer 2: I don't think you need to capitalize since you're not bolding the actual field names from the UI
